### PR TITLE
More updates for Sydney

### PR DESCRIPTION
**Pull request by aaronpowell**

### CODE REVIEW

File: _events/2019/sydney.md [hunk 1]
```diff
@@ -32,20 +32,24 @@ host: aaron_powell
 speakers:
   amy_kapernick:
     talk: "So You Think You Know HTML?"
+    place: "champion"
```
Comment (toddhgardner) [inline _events/2019/sydney.md]: She hasn't won yet. This is for tagging the  finalists after the event.

File: _events/2019/sydney.md [hunk 2]
```diff
@@ -32,20 +32,24 @@ host: aaron_powell
...
   jennifer_wadella:
     talk: ""
   richard_campbell:
-    talk: ""
+    talk: "Goliath."
   heather_downing:
     talk: ""
+  rob_crowley:
+    talk: "10x from the trenches"
+    place: "finalist"
```
Comment (toddhgardner) [inline _events/2019/sydney.md]: Same, this is for after the event.

File: _events/2019/sydney.md (hunk 1 above)
Comment (aaronpowell) [reply _events/2019/sydney.md]: Oh right, I thought it was for tagging of previous events.

Can we tag people who've previously won?

Comment (toddhgardner) [reply _events/2019/sydney.md]: Should be done already. If you click on the speaker name, it shows you their history. Like this:
https://pubconf.io/speakers/amy_kapernick/

Comment (aaronpowell) [reply _events/2019/sydney.md]: Ah cool, so it is.

I should actually look at how the codebase works 😂

**Pull request merged**